pkl-core/src/test/kotlin/org/pkl/core/runtime/VmValueRendererTest.kt

```diff
@@ -32,4 +32,12 @@ class VmValueRendererTest {
     val none = VmNull.withDefault("default")
     assertThat(renderer.render(none)).isEqualTo("null")
   }
+
+  @Test
+  fun `render bytes`() {
+    val renderer = VmValueRenderer.singleLine(5000)
+    val bytes = VmBytes((-128..127).map { it.toByte() }.toByteArray())
+    assertThat(renderer.render(bytes))
+      .isEqualTo("Bytes(128, 129, 130, 131, 132, 133, 134, 135, ... <248.b more bytes>)")
```

Comment:
This looks wonky with the data size unit included:
```suggestion
      .isEqualTo("Bytes(128, 129, 130, 131, 132, 133, 134, 135, ... <248 more bytes>)")
```

Reply:
Ah, yeah. This might be megabytes, or gigabytes, though.

I think maybe a representation like this makes sense:

```
Bytes(0, 0, 0, 0, 0, 0, 0, 0, ... <total size: 135.b>)
Bytes(0, 0, 0, 0, 0, 0, 0, 0, ... <total size: 2.15.gb>)
Bytes(0, 0, 0, 0, 0, 0, 0, 0, ... <total size: 3.mb>)
```

Reply:
Yeah, I think that's a better compromise here!